### DISCUSSION TITLE
Works like a charm. Why not post to AppStore

### DISCUSSION THREAD

**Discussion by pranaynaiduveera** [ther0n/UnnaturalScrollWheels#58] [General]
The app is working like a charm and really boosts the confidence that it is opensource. Why not post it to AppStore so that it can be discovered much better..?

**Comment by davidleejy** [comment]
I'm no MacOS/iOS app developer but the maintainers might consider monetizing this app via the app store since Apple has, for some time, held firmly to the belief that the scroll direction (and other cursor-related options) should be shared across quite dissimilar peripheral devices (e.g., trackpad and mouse) 🤷🏻. Monetizing will (i) help compensate the developer(s) of this app for their time and effort, and (ii) offer a simpler venue of installation for non-power users unfamiliar with brew or the terminal, like my sweet old neighbour who, bless her heart, has to deal with this silliness.
